Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/PressableButtons/PressableButton.cs

```diff
@@ -60,7 +60,7 @@ public class PressableButton : MonoBehaviour, IMixedRealityTouchHandler
         // The maximum distance before the button is reset to its initial position when retracting.
         private const float MaxRetractDistanceBeforeReset = 0.0001f;
 
-        private float currentPushDistance = 0.0f;
+        protected float currentPushDistance = 0.0f;
```

Comment:
This actually seems like something that would be useful to expose as public property (getter only) rather than make protected. For example for a script that wants to animate something based on pressed amount.

Reply:
Made it public. Thanks!